.changeset/thirty-mails-crash.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ponder/core": patch
+---
+
+Improved abi decoding performance by avoiding extra event selector calculation
```

Comment:
Improved ABI decoding performance, leading to a ~10% improvement in historical indexing speed.